frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +244,25 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbs = (modelRef, kindObj, match) => {
+  const model = modelFor(modelRef);
+  if (!match.params.ns) {
+    match.params.ns = sessionStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
```

Comment:
We shouldn't be overriding any values in `match`. I see why you're doing it, but that value is set by react-router should be treated as immutable. I'm not sure what consequences changes it could have. I would skip calling `getBreadcrumbPath` and just build the path yourself.

Reply:
@spadgett fixed, used `resourceListPathFromModel` with `lastBreadcrumb` to determine correct breadcrumb path.